src/main/java/ita/softserve/course_evaluation_admin/repository/CourseRepository.java

```diff
@@ -10,4 +12,15 @@
     @Query(value = "SELECT c.id AS id, c.course_name AS course_name, c.description AS description, c.start_date AS start_date, c.end_date AS end_date, " +
             "c.teacher_id AS teacher_id FROM course c INNER JOIN course_group cg ON c.id = cg.course_id WHERE cg.group_id = ?1", nativeQuery = true)
     List<Course> findAllByGroupId(long id);
+
+    @Query(value = "SELECT DISTINCT c.id AS id, c.course_name AS course_name, c.description AS description, c.start_date AS start_date, c.end_date AS end_date, c.teacher_id AS teacher_id FROM course c " +
```

Comment:
What is the reason to duplicate the column name in your query as the same alias? (for example:  c.course_name AS course_name)

Reply:
fixxed